hack/run-ci-e2e-test.sh

```diff
@@ -35,7 +35,8 @@ OSDK_WMCO_test() {
   fi
 }
 
-while getopts ":n:k:b:s" opt; do
+TEST="all"
+while getopts ":n:b:st:" opt; do
```

Comment:
should there be a : between s and t?

Reply:
Not from my reading of `man getopts`. Please double check and let me know.